frontend/src/pages/Scan/Scan.tsx

```diff
@@ -0,0 +1,245 @@
+import React, { useEffect, useState, useCallback } from 'react';
+import { useParams } from 'react-router-dom';
+import { useAuthContext } from 'context';
+import classes from './styles.module.scss';
+import { Organization as OrganizationType, Scan, ScanSchema } from 'types';
+import {
+  Header,
+  Button,
+  Form,
+  Dropdown,
+  TextInput,
+  Checkbox,
+  Label
+} from '@trussworks/react-uswds';
+import MultiSelect from '../Scans/MultiSelect';
+import { OrganizationOption } from 'pages/Scans/ScansView';
+import { Link } from 'react-router-dom';
+
+interface Errors extends Partial<OrganizationType> {
+  global?: string;
+}
+
+const ScanComponent: React.FC = () => {
+  const { scanId } = useParams();
+  const { apiGet, apiPut } = useAuthContext();
+  const [scan, setScan] = useState<Scan>();
+  const [errors, setErrors] = useState<Errors>({});
+  const [message, setMessage] = useState<string>('');
+  const [organizationOptions, setOrganizationOptions] = useState<
+    OrganizationOption[]
+  >([]);
+  const [scanSchema, setScanSchema] = useState<ScanSchema>({});
+  const [values, setValues] = useState<{
+    name: string;
+    organizations: OrganizationOption[];
+    arguments: string;
+    frequency: number;
+    frequencyUnit: string;
+    isGranular: boolean;
+    isSingleScan: boolean;
+  }>({
+    name: 'scan.name',
+    arguments: '{}',
+    organizations: [],
+    frequency: 1,
+    frequencyUnit: 'minute',
+    isGranular: false,
+    isSingleScan: false
+  });
+
+  const fetchScan = useCallback(async () => {
+    try {
+      const { scan, schema, organizations } = await apiGet<{
```

Comment:
I'm getting this error when clicking on the "Edit" button for a scan:

![image](https://user-images.githubusercontent.com/1689183/96325709-0c717400-0ff8-11eb-8703-ed80616e1f79.png)


This is likely because the actual response from this endpoint is structured a bit differently:

![image](https://user-images.githubusercontent.com/1689183/96325715-172c0900-0ff8-11eb-8aac-63abf310dd61.png)